assets/schema_input.json

```diff
@@ -15,13 +15,13 @@
             },
             "file": {
                 "type": "string",
-                "pattern": "^\\S+\\.(bam)|((vcf|bcf)(\\.gz))?$",
-                "errorMessage": "BAM, VCF or BCF file must be provided, cannot contain spaces and must have extension '.bam' or '.vcf', '.bcf' with optional '.gz' extension"
+                "pattern": "^\\S+\\.(bam|cram)|((vcf|bcf)(\\.gz))?$",
+                "errorMessage": "BAM, CRAM, VCF or BCF file must be provided, cannot contain spaces and must have extension '.bam', '.cram' or '.vcf', '.bcf' with optional '.gz' extension"
             },
             "index": {
-                "errorMessage": "Input file index must be provided, cannot contain spaces and must have extension '.bai', '.tbi' or '.csi'",
+                "errorMessage": "Input file index must be provided, cannot contain spaces and must have extension '.bai', 'crai', '.tbi' or '.csi'",
```

Comment:
Oooh,
so index could be bam/cram index or vcf index

Reply:
Yeah should we test that the user correctly pass the expected index depending on the file ?